src/reducers/account/claims.ts

```diff
@@ -1,26 +1,55 @@
 import { AnyAction } from 'redux'
-import { Map } from 'immutable'
+import Immutable from 'immutable'
+import { ClaimsState, CategorizedClaims } from 'src/reducers/account'
 
-const initialState = Map({
-  loading: false,
-  savedClaims: {
-    personal: [{
-      claimField: 'name',
-      category: 'personal'
+let categorizedClaims: CategorizedClaims = {
+  'Personal' : [{
+      displayName: 'Name',
+      type: ['Credential', 'ProofOfNameCredential'],
+      claims: [{
+        id: 'default1',
+        name: 'name',
+        value: 'name',
```

Comment:
The value field needs to be empty in initial state so that the user sees that he has to add it.